dataquality/integrations/seq2seq/hf.py

```diff
@@ -66,7 +66,7 @@ def watch(
     model: PreTrainedModel,
     tokenizer: PreTrainedTokenizerFast,
     generation_config: GenerationConfig,
-    generate_training_data: bool = False,
+    generatation_splits: List[str] = [],
```

Comment:
I'd recommend doing something like:
```suggestion
    generatation_splits: Optional[List[str]] = None,
    ...
    generatation_splits = generatation_splits or []
```

Mutable objects!